tools/integration_tests/read_cache/setup_test.go

```diff
@@ -0,0 +1,97 @@
+package read_cache
+
+import (
+	"log"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/internal/config"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/mounting/dynamic_mounting"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/mounting/only_dir_mounting"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/mounting/static_mounting"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
+)
+
+const (
+	testDirName           = "ReadCacheTest"
+	onlyDirMounted        = "Test"
+	cacheSubDirectoryName = "gcsfuse-file-cache"
+)
+
+var (
+	testDirPath       string
+	cacheLocationPath string
+	mountFunc         func([]string) error
+	// mount directory is where our tests run.
+	mountDir string
+	// root directory is the directory to be unmounted.
+	rootDir string
+)
+
+func createConfigFile() string {
+	cacheLocationPath = path.Join(setup.TestDir(), "cache-dir")
+
+	// Set up config file for file cache.
+	mountConfig := config.MountConfig{
+		FileCacheConfig: config.FileCacheConfig{
+			// Keeping the size as low because the operations are performed on small
+			// files
+			MaxSizeInMB: 10,
```

Comment:
How small are the files that we are testing. I think we would want to test random and sequential both. For Random itself, it gets only if > 8MB etc. So 10MB looks too low IMO

Reply:
This config file is only for sequential reads where we are planning to test only for small files. We also have plans to test with bigger files for range reads (both sequential and random) which will have a separate config file. I'll make the required parameters configurable in the function.

Reply:
Sounds good.